### PR TITLE
GoldToken & CeloRegistry to L2

### PR DESCRIPTION
**Pull request by pahor167**
Introduces GoldToken and CeloRegistry to L2 genesis block.

CeloRegistry 0x4200000000000000000000000000000000000022
GoldToken 0x4200000000000000000000000000000000000023

### CODE REVIEW

File: packages/contracts-bedrock/lib/ds-test
Comment (karlb) [inline packages/contracts-bedrock/lib/ds-test:1]: What is the purpose of this dependency? I don't see it used anywhere?

File: packages/contracts-bedrock/deploy-config/devnetL1.json [hunk 1]
```diff
@@ -41,10 +41,10 @@
...
   "eip1559Elasticity": 6,
-  "l1GenesisBlockTimestamp": "0x64c811bf",
+  "l1GenesisBlockTimestamp": "0x6502e1ca",
   "l2GenesisRegolithTimeOffset": "0x0",
   "faultGameAbsolutePrestate": "0x41c7ae758795765c6664a5d39bf63841c71ff191e9189522bad8ebff5d4eca98",
   "faultGameMaxDepth": 30,
   "faultGameMaxDuration": 300,
   "systemConfigStartBlock": 0
-}
+}
```
Comment (karlb) [inline packages/contracts-bedrock/deploy-config/devnetL1.json:50]: Strange that it does not generate the final newline as before. Not really a problem, though.

File: op-bindings/bindings/addressmanager.go
Comment (karlb) [inline op-bindings/bindings/addressmanager.go:1]: Why do we have changes in all generated bindings? Do we use a different version of `abigen`? I would be good to avoid unnecessary changes to reduce the amount of merge conflicts we get when rebasing on a more recent upstream version.

File: cel2-testnet/generated/genesis-l2.json
Comment (karlb) [inline cel2-testnet/generated/genesis-l2.json:1]: I didn't really think about when we want to update this, but my initial reaction is to only update it when we restart a testnet from scratch, so that we have the genesis available that is currently in use in the testnet.

File: op-chain-ops/genesis/config.go [hunk 2]
```diff
@@ -734,6 +734,19 @@ func NewL2StorageConfig(config *DeployConfig, block *types.Block) (state.Storage
 		"_initialized":  2,
 		"_initializing": false,
 	}
+	storage["GoldToken"] = state.StorageValues{
+		"initialized":  true,
```
Comment (karlb) [inline op-chain-ops/genesis/config.go:738]: Cool that they map the variable names here, so that we don't have to use the memory addresses!

File: cel2-testnet/generated/genesis-l2.json
Comment (pahor167) [reply cel2-testnet/generated/genesis-l2.json:1]: Since we are adding new predeploys, it felt like we should have this updated. But we can definitely try to setup process for this.

File: op-chain-ops/genesis/config.go (hunk 2 above)
Comment (pahor167) [reply op-chain-ops/genesis/config.go:738]: Agreed ! It took some digging to figure out whether and how they support mappings :-)

File: op-bindings/bindings/addressmanager.go
Comment (pahor167) [reply op-bindings/bindings/addressmanager.go:1]: Incorrect abigen version - fixed

File: packages/contracts-bedrock/lib/ds-test
Comment (pahor167) [reply packages/contracts-bedrock/lib/ds-test:1]: Good catch - deleted

**Review by karlb — APPROVED** [review]
